src/js/view/controls/controlbar.js

```diff
@@ -197,6 +197,9 @@ export default class Controlbar {
         const nextElement = elements.next.element();
         const nextUpTip = SimpleTooltip(nextElement, 'next', localization.nextUp, () => {
             const nextUp = _model.get('nextUp');
+            if (!nextUp) {
+                return;
+            }
```

Comment:
Exit early if `nextUp` is undefined, otherwise an exception will be thrown below when trying to trigger "nextShown" with `feedData: nextUp.feedData`.